api/app/Policies/PoolPolicy.php

```diff
@@ -75,40 +76,48 @@ public function viewAnyPublished(?User $user)
      */
     public function create(User $user, $request)
     {
-        if (array_key_exists('team_id', $request)) {
-            $team_id = $request['team_id'];
 
-            // Get the team to check against
-            $team = Team::find($team_id);
+        $teamId = isset($request['team_id']) ? $request['team_id'] : null;
+        $communityId = isset($request['community_id']) ? $request['community_id'] : null;
 
-            // Confirm the user can create pools for the team
-            if (! is_null($team)) {
-                if ($user->isAbleTo('create-team-draftPool', $team)) {
-                    return true;
-                }
-            } else {
-                return Response::deny('Cannot find a team matching team_id.');
-            }
-        } else {
-            Response::deny('Pool must be associated with a team when it is created.');
+        if (is_null($teamId) || is_null($communityId)) {
+            return false;
+        }
+
+        if ($user->isAbleTo('create-any-pool')) {
+            return true; // return early, permission does not exist at the moment
         }
```

Comment:
Added per

![image](https://github.com/user-attachments/assets/6785a4fd-a6cc-4707-b1c4-18bebd1d2d89)

Does not exist

![image](https://github.com/user-attachments/assets/0a8ab0c7-9b6e-4bb3-836f-e369daba07c2)

And neither does `create-any-draftPool`

![image](https://github.com/user-attachments/assets/e90c39cf-ca8c-4b4f-b878-d9d9586f13cf)

Only the team variant exists

![image](https://github.com/user-attachments/assets/bda1158e-d84b-4184-940e-bd2c82e903e5)